lib/pilot/options.rb

```diff
@@ -75,8 +75,16 @@ def self.available_options
                                      is_string: false,
                                      verify_block: proc do |value|
                                        raise "Please enter a valid positive number of seconds" unless value.to_i > 0
-                                     end)
-
+                                     end),
+        FastlaneCore::ConfigItem.new(key: :build_full_version,
+                                     short_option: "-b",
+                                     env_name: "PILOT_BUILD_FULL_VERSION",
+                                     description: "Build version with format version(build_number), ex 1.0(12)",
+                                     is_string: true,
+                                     optional: true,
+                                     verify_block: proc do |value|
+x                                       raise "Please use valid format, ex 1.0(12)" unless (value =~ /.*\(.*\)/)
+                                       end)
```

Comment:
`end` at 87, 39 is not aligned with `proc do |value|` at 85, 51 or `verify_block: proc do |value|` at 85, 37.